actors/evm/src/interpreter/instructions/call.rs

```diff
@@ -120,9 +122,15 @@ pub fn call<RT: Runtime>(
         };
 
         if precompiles::Precompiles::<RT>::is_precompile(&dst) {
+            let context = PrecompileContext {
+                is_static: matches!(kind, CallKind::StaticCall),
```

Comment:
We may need to check if we're in readonly mode as well?

Reply:
definitely.

Reply:
`matches!(kind, CallKind::StaticCall) || system.readonly`?